### DISCUSSION TITLE
Feedback on the Helm chart for creating a Selenium Grid Server in Kubernetes

### DISCUSSION THREAD

**Discussion by VietND96** [SeleniumHQ/docker-selenium#2202] [Ideas]
We would like to hear your feedback on the Helm chart for creating a Selenium Grid Server in Kubernetes.
Does it meet your expectations to simplify the deployment?
Is there anything that is not good or needs improvement? Is there a new idea or feature that you want the chart to support?

**Comment by rajat2911** [comment]
Hello @VietND96 We found out some issues while upgrading selenium-grid to v0.30.0
The selenium-grid-patch-scaledobjects-finalizers pod is having some issues due to which the deployment is not coming up, can see errors in logs: error: no objects passed to apply. 

Helm Errors : 
`STDERR:
  coalesce.go:289: warning: destination for selenium-grid.nodeConfigMap.extraScripts.nodePreStop.sh is a table. Ignoring non-table value ()
  coalesce.go:289: warning: destination for selenium-grid.nodeConfigMap.extraScripts.nodeProbe.sh is a table. Ignoring non-table value ()
  coalesce.go:289: warning: destination for selenium-grid.nodeConfigMap.extraScripts.nodeProbe.sh is a table. Ignoring non-table value ()
  coalesce.go:289: warning: destination for selenium-grid.nodeConfigMap.extraScripts.nodePreStop.sh is a table. Ignoring non-table value ()
  Error: UPGRADE FAILED: post-upgrade hooks failed: 1 error occurred:
  	* timed out waiting for the condition
COMBINED OUTPUT:
  coalesce.go:289: warning: destination for selenium-grid.nodeConfigMap.extraScripts.nodePreStop.sh is a table. Ignoring non-table value ()
  coalesce.go:289: warning: destination for selenium-grid.nodeConfigMap.extraScripts.nodeProbe.sh is a table. Ignoring non-table value ()
  coalesce.go:289: warning: destination for selenium-grid.nodeConfigMap.extraScripts.nodeProbe.sh is a table. Ignoring non-table value ()
  coalesce.go:289: warning: destination for selenium-grid.nodeConfigMap.extraScripts.nodePreStop.sh is a table. Ignoring non-table value ()
  Error: UPGRADE FAILED: post-upgrade hooks failed: 1 error occurred:
  	* timed out waiting for the condition`
